packages/block-library/src/button/edit.js

```diff
@@ -35,6 +35,16 @@ import { useMergeRefs } from '@wordpress/compose';
 import { prependHTTP } from '@wordpress/url';
 
 const NEW_TAB_REL = 'noreferrer noopener';
+const NEW_TAB_TARGET = '_blank';
+const NOFOLLOW_REL = 'nofollow';
+
+const LINK_SETTINGS = [
```

Comment:
Note to self that we should expose these settings as non-defaults from the component.

Reply:
I moved those values to /utils file